packages/web/components/indicators/pill.tsx

```diff
@@ -0,0 +1,7 @@
+import React, { FC } from "react";
+
+export const Pill: FC<{ children: React.ReactNode }> = ({ children }) => (
+  <div className="inline-block flex w-16 items-center justify-center rounded-full bg-gradient-positive px-2 py-1 uppercase text-black">
```

Comment:
I'm getting a lint error that:

`'inline-block' applies the same CSS properties as 'flex'.(cssConflict)`

Do you have prettier and/or eslint extension in VSCode?

Reply:
lint runs as a pre-commit hook, so it must be broken